bin/modbus

```diff
@@ -45,17 +77,33 @@ def main():
     parser.add_argument('-p', '--stop-bits', type=int, default=1)
     parser.add_argument('-P', '--parity', choices=['e', 'o', 'n'], default='n')
     parser.add_argument('-v', '--verbose', action='store_true')
+    parser.add_argument('-t', '--timeout', type=float, default=5.0)
     parser.add_argument('device')
     parser.add_argument('access', nargs='+')
     args = parser.parse_args()
 
-    if args.verbose:
-        logging.basicConfig(format='\033[90m%(message)s\033[39m', level=logging.DEBUG)
+    clr.init()
+    try:
+
+        mainLogger = logging.getLogger()            # Main logger
+
+
+
+        if args.verbose:
+            mainLogger.setLevel(logging.DEBUG)
+        else:
+            mainLogger.setLevel(logging.INFO)
+
+        ch = ColourHandler()
+        mainLogger.addHandler(ch)
 
-    definitions = Definitions()
-    definitions.parse(args.registers + os.environ.get('MODBUS_DEFINITIONS', '').split(':'))
+        definitions = Definitions()
+        definitions.parse(args.registers + os.environ.get('MODBUS_DEFINITIONS', '').split(':'))
 
-    connect_to_device(args).perform_accesses(parse_accesses(args.access, definitions), definitions).close()
+        connect_to_device(args).perform_accesses(parse_accesses(args.access, definitions), definitions).close()
 
+    finally:
```

Comment:
This looks like a job for a context manager, if `colorama` has one. Otherwise this will work :)

Reply:
They [don't seem to have one](https://github.com/tartley/colorama/blob/master/colorama/__init__.py).

On the other hand, I've used colorama on windows a bunch without calling `deinit()`, and it didn't seem to have any negative side-effects, but the documentation says call `deinit()`, so I'm calling `deinit()`